### PR TITLE
Add routes for /apps and render a placeholder

### PR DESCRIPTION
**Pull request by oponder**
This sets up the `/apps` routes and adds `Apps` to the navigation.

I keep the `/app-catalog` routes in this PR and will remove them all and the old components in a separate PR, probably the last one.

This gets merged into a feature collection branch here: https://github.com/giantswarm/happa/pull/2007
Towards: giantswarm/giantswarm#13652

### CODE REVIEW

File: src/components/Apps/Apps.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { Breadcrumb } from 'react-breadcrumbs';
+import { connect } from 'react-redux';
+import { Route, Switch } from 'react-router-dom';
+import { AppsRoutes } from 'shared/constants/routes';
+
+class Apps extends React.Component {
```
Comment (axbarsan) [inline src/components/Apps/Apps.js:7]: Not sure what your plans are for this file, but does it have to be JavaScript?

Comment (oponder) [reply src/components/Apps/Apps.js:7]: We can make it typescript, but it'd be the first of these routing components I think.

Wasn't sure if I should break the mold already.

Comment (axbarsan) [reply src/components/Apps/Apps.js:7]: Let's do it! 👍🏻

File: src/components/Apps/Apps.tsx [hunk 2]
```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { Breadcrumb } from 'react-breadcrumbs';
+import { Route, Switch } from 'react-router-dom';
+import { AppsRoutes } from 'shared/constants/routes';
+
+const Apps: React.FC = () => {
+  // eslint-disable-next-line class-methods-use-this
```
Comment (axbarsan) [inline src/components/Apps/Apps.tsx]:
```suggestion
```

**Review by axbarsan — APPROVED** [review]
LGTM

**Pull request merged**